test-network-function/turnium/turnium.json

```diff
@@ -0,0 +1,23 @@
+{
+	"identifier": {
+	  "url": "http://test-network-function.com/tests/turnium",
+	  "version": "v1.0.0"
+	},
+	"description": "Check `legids -v` reports state as up.",
+	"testResult": 0,
+	"testTimeout": 10000000000,
+	"reelFirstStep": {
+	  "execute": "legids -v\n",
+	  "expect": [
+		"state: up"
```

Comment:
We should probably add logic for `state: down`.  After all, state down implies a fail rather than an error (which is the default test result).